assets/src/blocks/Cookies/CookiesFrontend.js

```diff
@@ -155,7 +155,7 @@ export const CookiesFrontend = props => {
                 editable={isEditing}
                 allowedFormats={[]}
               />
-              <span className="always-enabled">{__('Always enabled', 'planet4-master-theme')}</span>
+              <span className="always-enabled">{__('Always enabled', 'planet4-blocks-backend')}</span>
```

Comment:
I think this one should not use the backend domain 🤔

Reply:
Yes, you are right 👍